flight-reservation/App.js

```diff
@@ -16,5 +16,6 @@ const styles = StyleSheet.create({
     backgroundColor: '#fff',
     alignItems: 'center',
     justifyContent: 'center',
+    paddingTop: Platform.OS === "android" ? StatusBar.currentHeight : 0,
```

Comment:
The styles are placed in an external file.

Reply:
Addressed (:

Reply:
App.js will be updated when we implement the StackNavigations components. Then it would not contain styles at the end. Should we remove it now?